cURL/pdf-with-added-image.sh

```diff
@@ -0,0 +1,10 @@
+curl -X POST "https://api.pdfrest.com/pdf-with-added-image" \
```

Comment:
Please do `chmod u+x pdf-with-added-image.sh` so this has permissions to run right off the bat.